### DISCUSSION TITLE
G7 and smartwatch

### DISCUSSION THREAD

**Discussion by Sandra7000** [NightscoutFoundation/xDrip#3008] [Q&A]
I really enjoyed the possibility of connecting my G6 directly with my (BlueJay) watch using xDrip+. Now I switched to a G7 sensor. No problem to connect my phone to the G7 but what about the BlueJay (or any other) watch? Is there any possibility that the G7 connects directly to the watch? Is there an update available of the xDrip Core (or will be available in the near future) to install on the watch? Or is it a problem of the G7 sensor, even though it can be read by two devices at the same time?
Does anybody know? 
Thanks for your help!

**Comment by belfastraven** [comment]
@jamorham   would more have to be done to allow sending config from phone, other than enabling 4 digit code as a valid id?  I know that the code is not public,  but I'd be happy to look at it.  Is it some kind of C or C++?   I really count on being able to use the watch as a collector so that I don't have to always carry my phone.     I normally run the bluejay on the phone slot,  since I have to have a receiver for insurance purposes.

**Reply by Sandra7000** [reply]
@belfastraven It would be really great if you had a solution for connecting the G7 to the BlueJay watch directly. I only noticed how great it is monitoring my bc without having my mobile phone with me all the time at the moment I changed to the G7. As I am a sports person and as I missed this feature so much, I meanwhile switched back to G6. However, if there was a solution for this issue, I would love to use the smaller G7 again. Thanks for your offer to help.

**Comment by belfastraven** [comment]
Just got a reply from bluejay support,  where I had inquired about this,
too.

Hi Marti

The current watch model GTS does not have the capacity to decode the signal
from the G7. We are working on a new model that will handle the G7, no firm
results yet, we will note your interest and update you with any progress.

Richard

**Reply by Sandra7000** [reply]
That's really good news. Thanks for sharing this info.
